client/src/snippets.json

```diff
@@ -125,14 +125,14 @@
     "Data parameter with single required dataset": {
         "prefix": "gx-data-single-param",
         "body": [
-            "<param argument=\"--$1\" type=\"data\" format=\"$2\" label=\"$3\" help=\"${4:TODO}\" />"
+            "<param argument=\"${1:--}\" name=\"$1\" type=\"data\" format=\"$2\" label=\"$3\" help=\"${4:TODO}\" />"
```

Comment:
```suggestion
            "<param argument=\"--${1:}\" name=\"$1\" type=\"data\" format=\"$2\" label=\"$3\" help=\"${4:TODO}\" />"
```
I guess what you really want is something like this?
![snippet_fix](https://user-images.githubusercontent.com/46503462/135416966-501845a8-8e8f-4041-8f0b-ea056e6b8bed.gif)

Reply:
Yes, thanks!